pkg/query-service/tests/test-deploy/docker-compose.yaml

```diff
@@ -38,17 +38,19 @@ services:
     image: signoz/query-service:latest
     container_name: query-service
     command: ["-config=/root/config/prometheus.yml"]
+    ports:
+      - "8180:8080"
```

Comment:
@prashant-shahi any reason for a different port mapping?

Reply:
Not sure. It was always like that for go testing.

Likely because contributors might be running the binary locally and already exposing `8080` port, when trying to perform `go test`.